src/pages/Chat/Chat.tsx

```diff
@@ -72,17 +72,18 @@ const Chat: React.FC<ChatProps> = ({
     newChat.startMessageFetchProcess(topic);
     console.info("Message fetch process started.");
     newChat.startUserFetchProcess(topic);
+    // probably move this more down
 
     // Connect to chat
-    await newChat
+    /*await newChat
```

Comment:
Will it work without this? If so, maybe it can be removed to avoid confusion.

Reply:
It will work, difference is, that the user is not connected for writing, only reading messages.